profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/tmgmt_poetry.module

```diff
@@ -225,6 +226,20 @@ function tmgmt_poetry_form_tmgmt_job_form_alter(&$form, &$form_state, $form_id)
 }
 
 /**
+ * Custom validator to add a server validation step for dates in the past.
+ */
+function _tmgmt_poetry_validate($form, &$form_state) {
```

Comment:
I think the function name should be more descriptive about its function. Something like "_tmgmt_poetry_validate_job_requested_date"